### PR TITLE
Add commit-message-convention into guide

### PR DESCRIPTION
**Pull request by tennisonchan**
Free feel to add more guideline for commit message :santa: :+1:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Guides for getting things done, programming well, and programming in style.
   * [Rails](/protocols/rails)
   * [Features](/protocols/features)
   * [Ember](/protocols/ember)
+* [Commit Message](/commit-message-convention)
```
Comment (toreriklinnerud) [inline README.md]: Indentation

Comment (tennison35) [inline README.md]: Oops! Thanks :) Try but cannot review markdown in python simplehttpserver. Will learn how to set it up later.

Comment (daemonsy) [inline README.md]: There's a preview button on github you know? :trollface:

Comment (tennison35) [inline README.md]: :sweat_smile: now I know. Thanks.

**Comment by matteodepalo** [comment]
:+1:

**Comment by tadast** [comment]
http://chris.beams.io/posts/git-commit

**Pull request merged**